src/lms-ds-loader/tests_integration_mssql/conftest.py

```diff
@@ -15,23 +19,85 @@
 from edfi_sql_adapter.sql_adapter import (
     Adapter,
     create_mssql_adapter_with_integrated_security,
+    create_mssql_adapter,
 )
 
+load_dotenv()
 
-def _new_mssql_adapter() -> Adapter:
-    # TODO: create method for injecting these values, for test
-    return create_mssql_adapter_with_integrated_security(
-        "localhost", "test_integration_lms_toolkit", 1433, False, False
+
+def pytest_addoption(parser):
```

Comment:
The build.py script has command arguments for passing values into the tests, to support the Harmonizer. But the same scripts should also work with the DS Loader integration tests. This forced me to add the parser rules here, otherwise executing the DS Loader integration tests from build.py fails. So now I've actually taken care of https://tracker.ed-fi.org/browse/LMS-437